web/docs/handbook/contributing.mdx

```diff
@@ -12,10 +12,10 @@ Want to contribute? Why not jump into our GitHub repo and:
 
 - [Sponsor Supabase](https://github.com/sponsors/supabase).
 - [Answer Discussions](https://github.com/supabase/supabase/discussions).
-- Submit an issue.
+- [Submit an issue](https://github.com/supabase/supabase/issue).
```

Comment:
this one depends on the specific repo I think. Which brings me to something that would  be useful for this page...

we could introduce a small selection of our repos here, maybe a link and one line explanation of each of supabase/supabase, supabase/supabase-js, and supabase/gotrue followed by "explore all repos" which links to https://github.com/orgs/supabase/repositories